tests/bugs/nested-switch.slang

```diff
@@ -3,7 +3,7 @@
 //TEST(compute):COMPARE_COMPUTE: -shaderobj
 //TEST(compute):COMPARE_COMPUTE:-vk -shaderobj
 //TEST(compute):COMPARE_COMPUTE:-cpu -shaderobj
-//DISABLE_TEST(compute):COMPARE_COMPUTE_EX:-wgpu
+//TEST(compute):COMPARE_COMPUTE:-wgpu
```

Comment:
I am not sure it does, but shouldn't this have `-shaderobj` like other lines?
```
//TEST(compute):COMPARE_COMPUTE:-wgpu -shaderobj
```

Reply:
That flag is no longer meaningful. Every test is `-shaderobj` now, and this flag should be removed from all tests.